src/main/kotlin/frc/robot/subsystems/superstructure/climb/ClimbSubsystem.kt

```diff
@@ -0,0 +1,52 @@
+package frc.robot.subsystems.superstructure.climb
+import com.revrobotics.CANSparkMaxLowLevel
+import org.ghrobotics.lib.commands.FalconCommand
+import org.ghrobotics.lib.mathematics.units.*
+import org.ghrobotics.lib.mathematics.units.nativeunit.*
+import org.ghrobotics.lib.motors.rev.FalconMAX
+import org.ghrobotics.lib.wrappers.FalconDoubleSolenoid
+import org.ghrobotics.lib.wrappers.FalconSolenoid
+
+object ClimbSubsystem {
+
+    class Climb(val climbHeight : SIUnit<Meter>): FalconCommand() {
```

Comment:
Can we put this climb command in a different file and suffix the name with `Command`?